pkg/controller/deployment/generic_deployment_controller.go

```diff
@@ -0,0 +1,98 @@
+package deployment
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/openshift/cert-manager-operator/pkg/operator/kubeclient"
+	configv1 "github.com/openshift/client-go/config/clientset/versioned/typed/config/v1"
+	"github.com/openshift/library-go/pkg/controller/factory"
+	"github.com/openshift/library-go/pkg/operator/apiserver/controller/workload"
+	"github.com/openshift/library-go/pkg/operator/events"
+	"github.com/openshift/library-go/pkg/operator/resource/resourceapply"
+	"github.com/openshift/library-go/pkg/operator/resource/resourcemerge"
+	"github.com/openshift/library-go/pkg/operator/resource/resourceread"
+	"github.com/openshift/library-go/pkg/operator/status"
+	"github.com/openshift/library-go/pkg/operator/v1helpers"
+	appsv1 "k8s.io/api/apps/v1"
+	"k8s.io/client-go/informers"
+	"k8s.io/klog/v2"
```

Comment:
order, here and everywhere else

Reply:
I ran `goimports -w (find . -type f -name '*.go' -not -path "./vendor/*")` and it didn't change anything. Could you please me what's a reliable command I could use to automatically sort them?

Reply:
unless you find something more clever, you need to do the first ordering manually. VSCode with the Go extension is somewhat capable of sorting the imports itself after the groups have been created.

Reply:
Let me push back on doing any import sorting manually. At least for now.

I'll spin up a discussion about this in our private channel.

Reply:
Actually, I have another idea - I turned on import grouping in my IDE. Let's see if it's any better now?